tapis-typescript/e2e/streams.spec.ts

```diff
@@ -0,0 +1,153 @@
+// Load e2e environment variables into process.env
+require('dotenv').config({ path: 'e2e.env' });
+
+import 'mocha';
+import { before } from 'mocha';
+import { 
+  Streams
+} from '../src';
+import { getToken, checkJsonError } from './utils';
+import { expect } from 'chai';
+import fetch from 'cross-fetch';
+
+const version = 5;
+
+const projectId = `E2E_TEST_PROJECT_${version}`;
+const siteId = `E2E_TEST_SITE_${version}`;
+const instId = `E2E_TEST_INSTRUMENT_${version}`;
```

Comment:
These should be retrieved from process.env with the values stored in e2e.env.

Reply:
Migrated ids to e2e.env (listed in e2e.env.sample.